src/pudl/transform/eia860.py

```diff
@@ -169,10 +169,7 @@ def _core_eia860__ownership(raw_eia860__ownership: pd.DataFrame) -> pd.DataFrame
             "utility_id_eia": "operator_utility_id_eia",
             "utility_name_eia": "operator_name",
             "state": "operator_state",
-            "owner_name": "utility_name_eia",
-            "owner_utility_id_eia": "utility_id_eia",
-            "owner_state": "state",
-            "owner_country": "country",
+            "owner_name": "owner_utility_name_eia",
```

Comment:
Is there a reason why we need or want for these renames to happen in the transform, rather than ensuring that the columns have the desired name in the initial spreadsheet mapping CSVs?

Reply:
Not sure where the initial spreadsheet mapping happens, but you mean so that `_raw_eia860__ownership` also has these renamed columns?

Reply:
No preference from my end.

Reply:
i think it would be at least marginally nicer to rename these in the mapping CSVs. it is in `package_data/eia860/column_maps/ownership.csv`